### DISCUSSION TITLE
XIAO ESP32S3(sense) With Zephyr(RTOS) | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1797] [Q&A]
# XIAO ESP32S3(sense) With Zephyr(RTOS) | Seeed Studio Wiki

XIAO ESP32S3(sense) With Zephyr(RTOS)

https://wiki.seeedstudio.com/xiao_esp32s3_zephyr_rtos/

**Comment by ThongNguyen182003** [comment]
1. I try run this command in ubuntu 24.04
west build -p always -b xiao_esp32s3 samples/hello_world
west flash
west espressif monitor

2. But response showing:  "invalid header 0xffffffff"
